pybossa/cache/task_browse_helpers.py

```diff
@@ -350,6 +350,9 @@ def validate_user_preferences(user_pref):
     valid_languages = valid_user_preferences.get('languages')
     valid_locations = valid_user_preferences.get('locations')
 
+    # Support old spelling of Türkiye
+    if valid_locations:
+        valid_locations.append('Turkey')
```

Comment:
Any reason to not maintain this under config?

Reply:
The locations in the config are sent to the frontend to be displayed to the user (see https://github.com/bloomberg/pybossa/blob/main/pybossa/view/projects.py#L1787 ) We want to support the old spelling only in the backend while sending the new spelling to the frontend. 

So we should edit the config to the new spelling, and add the old spelling manually in the backend before running validation. This way we don't need to maintain two lists in config, and we can easily deprecate the old spelling when the time comes.

Reply:
As discussed, we now have separate ticket to handle old data and do this code cleanup, it looks good to me. Thanks!